src/OpenSage.Game/Data/W3d/W3dCollectionPlaceholder.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.IO;
+using System.Numerics;
+using OpenSage.Data.Utilities.Extensions;
+
+namespace OpenSage.Data.W3d
+{
+    public sealed class W3dCollectionPlaceholder : W3dChunk
+    {
+        public override W3dChunkType ChunkType { get; } = W3dChunkType.W3D_CHUNK_PLACEHOLDER;
+
+        public uint Version { get; private set; }
+        public Vector3 TransformA { get; private set; }
+        public Vector3 TransformB { get; private set; }
+        public Vector3 TransformC { get; private set; }
+        public Vector3 TransformD { get; private set; }
+        public string Name { get; private set; }
+
+        public byte[] UnknownBytes { get; private set; }
+
+        internal static W3dCollectionPlaceholder Parse(BinaryReader reader, W3dParseContext context)
+        {
+            return ParseChunk(reader, context, header =>
+            {
+                var result = new W3dCollectionPlaceholder
+                {
+                    TransformA = reader.ReadVector3(),
+                    TransformB = reader.ReadVector3(),
+                    TransformC = reader.ReadVector3(),
+                    TransformD = reader.ReadVector3(),
+                    Name = reader.ReadFixedLengthString(W3dConstants.NameLength),
+                    UnknownBytes = reader.ReadBytes((int)context.CurrentEndPosition - (int)reader.BaseStream.Position)
```

Comment:
If you do more PRs, please could you add a space after casts?  i.e.

``` csharp
UnknownBytes = reader.ReadBytes((int) context.CurrentEndPosition - (int) reader.BaseStream.Position)
```

Don't worry for this PR though.

Reply:
I went through and updated everything.